### PR TITLE
cp-415: Fix width for meditation topic cards on mobile size screen

### PR DESCRIPTION
**Pull request by OlesiaTr**
https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/104642736/2c656293-48a5-4dc3-ad06-db4a8e2aa7f5

### CODE REVIEW

**Review by liza-veis — APPROVED** [review]

File: frontend/src/pages/meditation/components/meditation-sidebar/styles.module.scss [hunk 1]
```diff
@@ -45,6 +45,7 @@
   height: 100%;
   margin-top: 83px;
   overflow-y: scroll;
+  scrollbar-width: none;
```
Comment (liza-veis) [inline frontend/src/pages/meditation/components/meditation-sidebar/styles.module.scss]: do we need this?

**Review by liza-veis — CHANGES_REQUESTED** [review]

File: frontend/src/pages/meditation/components/meditation-sidebar/styles.module.scss (hunk 1 above)
Comment (OlesiaTr) [reply frontend/src/pages/meditation/components/meditation-sidebar/styles.module.scss]: we might not, if there won't be more than n amount of meditation topics, that fit into the screen, including rotated screen on mobile?

**Review by HIMDanny — DISMISSED** [review]

File: frontend/src/pages/meditation/components/meditation-sidebar/styles.module.scss (hunk 1 above)
Comment (liza-veis) [reply frontend/src/pages/meditation/components/meditation-sidebar/styles.module.scss]: I didn't get your question, but we need to show scrollbar

Comment (OlesiaTr) [reply frontend/src/pages/meditation/components/meditation-sidebar/styles.module.scss]: Ok, changed it!

**Review by v0ldemar01 — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**